### PR TITLE
Add replication agent to publish

### PR DESCRIPTION
**Pull request by znikolovski**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $ cat .gitignore
 # exclude
-*.jar
```
Comment (dbenge) [inline .gitignore:3]: you did not like my wildcard?  ok

Comment (znikolovski) [inline .gitignore:3]: I did – but I had to add a couple of jar files for the compaction hence the more forgiving filter :)

Regards,

Zoran Nikolovski
Lead Software Engineer
e: znikolovski@odecee.com.aumailto:firstname.lastname@odecee.com.au
m: +61 4 0801 1215
w: https://au.linkedin.com/in/zorannikolovski

Comment (znikolovski) [inline .gitignore:3]: I needed to allow for the compaction file jars hence the more forgiving filter :)

Comment (dbenge) [inline .gitignore:3]: cool, works for me

**Pull request merged**